Description: Remove from .cabal data files, missing in tarball
Author: Dmitry Bogatov
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: b/gitit.cabal
===================================================================
--- a/gitit.cabal
+++ b/gitit.cabal
@@ -39,21 +39,14 @@ author:              John MacFarlane
 maintainer:          jgm@berkeley.edu
 bug-reports:         http://github.com/jgm/gitit/issues
 stability:           experimental
-extra-source-files:  data/static/js/jquery-1.2.6.js
-                     data/static/js/jquery.hotkeys-0.7.9.js
-                     data/static/js/jquery-ui.core-1.6rc2.js
-                     data/static/js/jquery-ui.droppable-1.6rc2.js
-                     data/static/js/jquery-ui.draggable-1.6rc2.js
-                     data/static/js/jquery-ui.tabs-1.6rc2.js
 data-files:          data/static/css/screen.css, data/static/css/print.css,
                      data/static/css/ie.css, data/static/css/highlighting.css,
                      data/static/css/reset-fonts-grids.css,
                      data/static/css/custom.css,
                      data/static/img/logo.png, data/static/img/icons/feed.png,
                      data/static/img/icons/folder.png, data/static/img/icons/page.png,
-                     data/static/js/dragdiff.js, data/static/js/jquery-1.2.6.min.js,
-                     data/static/js/uploadForm.js, data/static/js/jquery-ui-combined-1.6rc2.min.js,
-                     data/static/js/jquery.hotkeys-0.7.9.min.js,
+                     data/static/js/dragdiff.js,
+                     data/static/js/uploadForm.js,
                      data/static/js/preview.js, data/static/js/search.js,
                      data/static/js/MathMLinHTML.js, data/static/js/footnotes.js,
                      data/static/robots.txt,
